packages/core/bootstrap/src/lib/modules/requester.ts

```diff
@@ -48,7 +48,13 @@ export class Requester {
       let response: AxiosResponse<T>
       const url = join(config.baseURL || '', config.url || '')
       try {
+        const startTime = process.hrtime()
+
         response = await axios(config)
+
+        const endTime = process.hrtime(startTime)
+        const milliseconds = Math.round(endTime[0] * 1000 + endTime[1] / 1000000)
+        response.headers['request-duration'] = milliseconds.toString()
```

Comment:
Hmm, it might be too much work, yeah. I'm just not a very big fan of dumping data into headers just for the purpose of us passing variables.

Let's keep using the headers, but would prefer if we make it more explicit that this is an EA header (e.g. `ea-request-duration`).